<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fantasy Cricket Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <!-- Title -->
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Fantasy Cricket Team Selection</string>
     </property>
    </widget>
   </item>

   <!-- Two tables side by side -->
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_tables">
     <!-- Left: Available players -->
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_left">
       <item>
        <widget class="QLabel" name="availableLabel">
         <property name="text">
          <string>Available Players</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="playerTable">
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- Right: Selected team -->
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_right">
       <item>
        <widget class="QLabel" name="teamLabel">
         <property name="text">
          <string>Selected Team</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="teamTable">
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>

   <!-- Bottom: Save button -->
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_bottom">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save Team</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
